### ISSUE TITLE
Can not enable in phpbb 3.2.1

### ISSUE THREAD

**Issue by yylzcom** [gothick/phpbb-ext-akismet#6]
I upload it to the ext file under my phpbb, however I can not see any activate link nor any info on the admin control panel of phpbb.

Is there any advice, or does it work with phpbb 3.2.1? 

Thanks and best regards!

**Comment by Galixte** [comment]
Hi @yylzcom,

all phpBB extensions downloaded from GitHub aren't ready for installation.

Each time, you have to consult this line: https://github.com/gothick/phpbb-ext-akismet/blob/master/composer.json#L2 it indicates the correct folder names these you have to create in the ```./ext/``` folder.

**Comment by yylzcom** [comment]
Thank you Galixte,

@Galixte 
Thanks for your tips!
It works now on Version 3.2.1.  I just can not understand why the official team does not put it into the extension databases. I think akismet is one of the best anti-spa (free)service in the world.

**Comment by gothick** [comment]
@yylzcom Partly that's my fault—I only develop this in my spare time, and as I'm a contract programmer, that spare time varies a *lot*. This means I can never guarantee to be working on the extension when it needs it, or put in the time it needs to be submitted to the official repository. (I've not even had the time recently to really think about what needs doing about that problem!)

Basically this is an extension I use on a forum I run, and I figured that having it "out there" in the world was more helpful than keeping the repository private.

Glad you got it working. I've got a couple of weeks off soon, so I may be doing a bit more polishing of my extensions (but I need to get a couple of the others working on 3.2 before I spend too much time on this one.)
